src/components/ChatLog.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import './ChatLog.css';
+import PropTypes from 'prop-types';
+import ChatEntry from './ChatEntry';
+
+const ChatLog = (props) => {
+  const getChatLogJSX = (props) => {
+    return props.entries.map((entry) => {
+      return (
+        <ChatEntry
+          // key={entry.id}
+          // id={entry.id}
```

Comment:
Minor style preference: just get rid of commented code like this, or at the least explain why you're leaving the commented code in.